### PR TITLE
Snap turn preference

### PR DESCRIPTION
**Pull request by howard-stearns**
"Comfort Mode" is now called "Snap Turn", is an Avatar basic checkbox preference instead of a Developer->Avatar menu item, and has the same on/off behavior (with checkbox) regardless of whether in HMD.

### CODE REVIEW

**Comment by howard-stearns** [comment]
https://app.asana.com/0/32622044445063/89739941402545

See interpretation of "leave it on by default" in ticket.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4150d318d318b3747e5d57f8b8d77df6ce958c17 available at: https://gist.github.com/a127e950f7b1cef165ac<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7112.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7112.dmg

**Comment by howard-stearns** [comment]
DNM until I restore the "only in HMD"-ness of the behavior, and change the name to "Snap Turn when in HMD"

**Comment by howard-stearns** [comment]
Above changes made. Should be good to go.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d70584b5a33dcbb1739e3b7d1e87332ee660378 available at: https://gist.github.com/a127e950f7b1cef165ac<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7112.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7112.dmg

File: interface/resources/qml/hifi/MenuOption.qml [hunk 1]
```diff
@@ -35,7 +35,6 @@ QtObject {
     readonly property string centerPlayerInView: "Center Player In View";
     readonly property string chat: "Chat...";
     readonly property string collisions: "Collisions";
-    readonly property string comfortMode: "Comfort Mode";
```
Comment (ZappoMan) [inline interface/resources/qml/hifi/MenuOption.qml]: I think this file is dead code... we should remove it. Not a merge blocker.

**Pull request merged**